SplunkforPaloAltoNetworks/bin/panContentPack.py

```diff
@@ -196,19 +198,27 @@ def main():
     # Get the sessionKey
     sessionKey = settings['sessionKey']
 
-    log(debug, "Begin get API key")
+    logger.debug("Begin get API key")
+
     # Get the API key from the Splunk store or from the device at hostname if no apikey is stored
     apikey = common.apikey(sessionKey, args[0], debug)
-
     device = pandevice.base.PanDevice(args[0], api_key=apikey)
-    device.refresh_system_info()
+    try:
+        systeminfo = device.refresh_system_info()
+        logger.debug(systeminfo)
+    except Exception as e:
+        logger.error("Error refreshing device: %s" % e)
```

Comment:
This seems to continue the program in the event of an error. Might need to `exit_with_error()` here.